### ISSUE TITLE
In a doc page, move the summary box to the top of the content

### ISSUE THREAD

**Issue by fbunoz** [c2corg/v6_ui#1878]
The summary field is intended to show a text another place than description text field.
Today, the summary is showed just over the description : https://www.camptocamp.org/routes/927599/fr/tete-de-l-auriac-l-aiguille-la-voie-des-rats-morts
It should better to put the summary just under the title, with a police size a little bit bigger than normal size : 

![design_summary](https://user-images.githubusercontent.com/3902555/33836924-25e0e520-de8b-11e7-9ea4-0da2ecd1d259.png)

**Comment by brunobesson** [comment]
Makes sense. I would even remove the green left bar and use something else for emphasis. Maybe a quick poll in the forums could give us good ideas?

**Comment by cbeauchesne** [comment]
not fan : summary is part of non-objective data, I prefer to keep it close to the reste of description.

**Comment by brunobesson** [comment]
Still, placing it separated from the description section, would you agree?
What about placing it before the history then?

**Comment by cbeauchesne** [comment]
agree, i don't have a strong pov on the best order, until the factual info stays on the top.

**Comment by cbeauchesne** [comment]
and yes, after thinking about it, putting above history is IMO a good idea

**Comment by brunobesson** [comment]
@fbunoz what is your opinion on this?

**Comment by fbunoz** [comment]
> what is your opinion on this?

I want the summary above (or aside) the data. The summary must contain a little text to give the main infos not given by the ratings (beauty, rock quality, homogeneity, celebrity, classical or not, ...), useful on a card (for now the summary is showed only in the home cards, but it is useful for route filter cards).
Like on many books, or web site : http://www.vttour.fr/topos/col-de-marcieu,439.html

**Comment by brunobesson** [comment]
I guess a poll on the forums would be a nice way to settle this. Who's willing to handle it?

**Comment by cbeauchesne** [comment]
ok, i handle it
